src/test/sock_tests.cpp

```diff
@@ -95,7 +95,7 @@ static void CreateSocketPair(int s[2])
 static void SendAndRecvMessage(const Sock& sender, const Sock& receiver)
 {
     const char* msg = "abcd";
-    constexpr size_t msg_len = 4;
+    constexpr ssize_t msg_len = 4;
```

Comment:
This is used also as 3rd argument to `strncmp()` which is of type `size_t`. I hope this change will not produce a warning for that. If it does, then a typecast would be necessary.